payments-core/src/main/java/com/stripe/android/view/PaymentAuthWebViewClient.kt

```diff
@@ -161,7 +162,8 @@ internal class PaymentAuthWebViewClient(
 
     // pre-defined return URLs
     private fun isPredefinedReturnUrl(uri: Uri): Boolean {
-        return "stripejs://use_stripe_sdk/return_url" == uri.toString()
+        return "stripejs://use_stripe_sdk/return_url" == uri.toString() ||
+            uri.toString().startsWith(DefaultReturnUrl.PREFIX)
```

Comment:
This ensures we recognize the return URL. Without this, we look for the PI/SI id in the URL (lines 151-155 above), but it wil be different than the original PI/SI for Link.